code/__HELPERS/mobs.dm

```diff
@@ -317,4 +317,11 @@ Proc for attack log creation, because really why not
 					. = 0
 					break mainloop
 	if(progbar)
-		qdel(progbar)
\ No newline at end of file
+		qdel(progbar)
+
+/proc/isabductor(A)
+	. = FALSE
+	if(ishuman(A))
+		var/mob/living/carbon/human/H = A
+		if(H.dna && istype(H.dna.species, /datum/species/abductor))
+			. = TRUE

```

Comment:
Why not make this proc atom and override on humans with checks for species type?